Makefile

```diff
@@ -0,0 +1,43 @@
+.PHONY: build-garage-headless build-garage-nvidia \
```

Comment:
i'm not sure this setup exactly works for interactive use cases. what will happen to all my data files if i'm using these containers to run an experiment?

by default, they will be destroyed when the container stops.

Reply:
That's not completely true, I bind the data folder in the container with the data folder in the garage workspace using the argument:
```
-v $(shell pwd)/data:/root/code/garage/data
```
However, I accept this won't work when the container runs anywhere else in the file system. I will add a variable to pass the path to bind to the data folder in the container, having the garage/data folder as default, and if there's no folder in the path, then only in that case the data files will be lost.

Reply:
in retrospect, something like this is actually the right thing to do